prometheus_client/core.py

```diff
@@ -2,11 +2,8 @@
 
 from .metrics import Counter, Enum, Gauge, Histogram, Info, Summary
 from .metrics_core import (
-    CounterMetricFamily, GaugeHistogramMetricFamily, GaugeMetricFamily,
-    HistogramMetricFamily, InfoMetricFamily, Metric, Sample,
-    StateSetMetricFamily, SummaryMetricFamily, UnknownMetricFamily,
-    UntypedMetricFamily,
-)
+    CounterMetricFamily, GaugeHistogramMetricFamily, GaugeMetricFamily, HistogramMetricFamily, InfoMetricFamily,
```

Comment:
This was changed from this before on style grounds, is there a standard here?

Reply:
I am not sure about the standard here but after manually changing the flake8 comments on each file, I applied the auto-formatter from pycharm which yielded this result.

Reply:
I don't think it's consistent with what's in the tox.ini

Reply:
Tox.ini has flake8 where we aren't ignoring ordering of imports. If we want to ignore that, please let me know.

Reply:
Let's just make sure everything is consistent

Reply:
Everything is consistent rn according to flake8.